Mage/Command/BuiltIn/CompileCommand.php

```diff
@@ -22,19 +22,43 @@
 class CompileCommand extends AbstractCommand
 {
     /**
+     * @var Compiler
+     */
+    private $compiler;
+
+    public function __construct()
+    {
+        $this->setCompiler(new Compiler());
+    }
+
+    public function setCompiler(Compiler $compiler)
```

Comment:
For the different IDE's that are using it. Some may need it and it may help other contributors.

And maybe someday we create a API documentation for the project :-)